pkg/util/helperfunctions.go

```diff
@@ -58,3 +61,12 @@ func GenerateHashFromName(name string) uint64 {
 	_, _ = hash.Write([]byte(name))
 	return hash.Sum64()
 }
+
+func SetCommonAnnotations(context context.Context, object client.Object) {
```

Comment:
Unused context - future proofing or an error?

Reply:
Error, used the context in the helper function at first.